### DISCUSSION TITLE
FreeBSD deprecates qmail port

### DISCUSSION THREAD

**Discussion by rduffner** [msimerson/Mail-Toaster-6#533] [General]
Wow:
https://cgit.freebsd.org/ports/commit/?id=50e361577186fdefb1be29cbe06f304186f7e02b

I assume, as MT6 does its own builds, this won't affect it that much. But it may start failing to build in the future, especially with later versions of the toolchain.

**Comment by msimerson** [comment]
The author there is referring to his own deprecated qmail *patch*.
